docs/specs/linters_and_formatters.md

```diff
@@ -39,7 +39,7 @@ are provided for developers to check style issues.
 > files only. We hope that linters for other languages will be online in the
 > near future.
 
-|  Linters   |           Source Extensions           | Apollo Command Used       |
+|  Linters   |           Source Extensions           | Usage                     |
```

Comment:
Better to make it center aligned here to keep consistency.

Reply:
> Better to make it center aligned here to keep consistency.

I tried. But It seems our mdfmt will automatically put it leftside.